### DISCUSSION TITLE
Is there a way to render the bullet point "-" to "•"?

### DISCUSSION THREAD

**Discussion by neuromaancer** [epwalsh/obsidian.nvim#310] [Q&A] [answered]
I am wondering if there is a way to render the bullet point "-" to "•"?

**Comment by epwalsh** [comment]
Not at the moment though it would be straight-forward to add support for this. Would you want to see "•" in addition to the "✓"? Because currently we just hide the "-".

**Reply by neuromaancer** [reply]
I think it would be nice to see "•" as the bullet point, because this is a straight way to say this line is a bullet point line.

**Reply by epwalsh** [reply]
Ok so in addition to the other markup like the "✓"? For example:

• ✓

**Reply by neuromaancer** [reply]
For me, "•" is enough, but I think it would be great if someone wanted to customize it by themselves to create like "• ✓". Thank you very much!

**Reply by epwalsh** [reply]
You can get this behavior by updating the `ui.checkboxes` section of your config to:

```lua
          [" "] = { char = "•", hl_group = "ObsidianTodo" },
          ["x"] = { char = "•", hl_group = "ObsidianDone" },
          [">"] = { char = "•", hl_group = "ObsidianRightArrow" },
          ["~"] = { char = "•", hl_group = "ObsidianTilde" },
```

**Reply by neuromaancer** [reply]
I feel like I didn't explain myself well, I meant to show the syntax "-" of the markdown bullet point as "•". This is also how the obsidian app presents outlines. I made the change you suggested and it doesn't achieve that effect. Thank you.

**Reply by epwalsh** [reply, marked as answer]
Ah, I see. Ok I've implemented that in #320.

**Reply by neuromaancer** [reply]
Thank you so much!
